### PR TITLE
updated drag and drop widget

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-03-01 at 08 55 07|screen shot 2016-03-01 at 09 08 36|
|screen shot 2016-03-01 at 08 55 16 2|screen shot 2016-03-01 at 08 53 34 2|
|screen shot 2016-03-01 at 08 55 19|screen shot 2016-03-01 at 08 53 32|

### CODE REVIEW

File: filer/private/sass/settings/_custom.scss [hunk 1]
```diff
@@ -47,6 +48,8 @@ $screen-mobile: 320px;
 $screen-tablet: 720px;
 $screen-desktop: 975px;
 
+$screen-filer: 810px;
```
Comment (vxsx) [inline filer/private/sass/settings/_custom.scss]: what is this value really?

Comment (lory87) [inline filer/private/sass/settings/_custom.scss]: I had to add a separate value for filer because height will break if I choose screen tablet.

Comment (vxsx) [inline filer/private/sass/settings/_custom.scss]: i was just thinking if we could come up with something that is a bit more descriptive than just `filer`

**Comment by vxsx** [comment]
:+1:

File: filer/private/sass/settings/_custom.scss (hunk 1 above)
Comment (lory87) [inline filer/private/sass/settings/_custom.scss]: I updated name to filer-widget

**Comment by vxsx** [comment]
:+1:

**Pull request merged**